README.md

```diff
@@ -1,8 +1,10 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+contact form project
 
 > Description the project.
+> This is simple portfolio project
```

Comment:
- [ ] Please add some content in the `Getting started` section like the procedures to clone and run your project locally in a machine. This helps other people who are interested in your project to know how to run your project 😃